include/swift/Option/Options.td

```diff
@@ -129,6 +129,10 @@ def driver_always_rebuild_dependents :
   Flag<["-"], "driver-always-rebuild-dependents">, InternalDebugOpt,
   HelpText<"Always rebuild dependents of files that have been modified">;
 
+def enable_experimental_dependencies :
+Flag<["-"], "enable-experimental-dependencies">, Flags<[FrontendOption, HelpHidden]>,
+HelpText<"When interface changes, be more selective">;
```

Comment:
Nitpick: I don't understand this help text.

Reply:
It is a bit terse, but I'm not sure exactly what would be needed to make it understandable--I'm too close. Could you suggest something?

Reply:
I don't think there's context in what the "interface" is, but even knowing that this means "a file's semantic and binary interface as presented to the rest of the module", it doesn't say what the compiler is going to be more selective *about.*

Reply:
Just pushed a fix. Hope it's better.